test/system/030-run.bats

```diff
@@ -14,8 +14,8 @@ load helpers
     # ...but check the configured runtime engine, and switch to crun as needed
     run_podman info --format '{{ .Host.OCIRuntime.Path }}'
     if expr "$output" : ".*/crun"; then
-        err_no_such_cmd="Error: executable file.* not found in \$PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found"
-        err_no_exec_dir="Error: open executable: Operation not permitted: OCI permission denied"
+        err_no_such_cmd="Error:.*executable file.* not found in \$PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found"
+        err_no_exec_dir="Error:.*open executable: Operation not permitted: OCI permission denied"
```

Comment:
Why not include the `crun:` here?
```diff
-        err_no_such_cmd="Error:.*executable file.* not found in \$PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found"
-        err_no_exec_dir="Error:.*open executable: Operation not permitted: OCI permission denied"
+        err_no_such_cmd="Error: crun: executable file.* not found in \$PATH: No such file or directory: OCI runtime attempted to invoke a command that was not found"
+        err_no_exec_dir="Error: crun: open executable: Operation not permitted: OCI permission denied"
```
It seems like a perfect opportunity to confirm that the new code works as expected.

Reply:
Because if others run the tests using other runtimes, then it will blow up.

Reply:
Look at the line immediately above :grin:

Reply:
Missed that.

Reply:
It's OK. CI is green. I'll add this to my list of minor tweaks to bunch up and commit once in a while.

/lgtm
/hold